scripts/00spinup.py

```diff
@@ -57,23 +62,18 @@
 # Initialization
 # ------------------------------------------
 # Arrays for total values (across all layers), with time as the first axis
-subsurface_storage = np.zeros((nt,))
-surface_storage = np.zeros((nt,))
+subsurface_storage = np.zeros(nt)
+surface_storage = np.zeros(nt)
 wtd = np.zeros((nt, nx, ny))
-et = np.zeros((nt,))
+et = np.zeros(nt)
 overland_flow = np.zeros((nt, nx, ny))
 
 # ------------------------------------------
 # Loop through time steps
 # ------------------------------------------
 for i, (pressure_file, saturation_file) in enumerate(zip(pressure_files, saturation_files)):
-    pressure = pfio.pfread(pressure_file)
-    saturation = pfio.pfread(saturation_file)
-
-    # We set pressure/saturation values outside our mask to np.nan to indicate missing values.
-    # Individual functions that deal with these arrays are written to handle np.nan values properly.
-    pressure[mask == 0] = np.nan
-    saturation[mask == 0] = np.nan
+    pressure = metadata.pfb_data(pressure_file)
```

Comment:
Instead the the script having to deal with `pfread`, if we obtain `pfb_data` directly from the `metadata` object, masking can be swept under the rug.